src/room.py

```diff
@@ -20,6 +20,14 @@ def __str__(self):
     def add_item(self, item):
         self.items.append(item)
 
+    def remove_item(self, item_name):
+        self.items.reverse()   # All the reverses are just to make it so you'll take the last item if there are duplicates
+        for i in range(len(self.items)):
```

Comment:
there's ways to do it without having to loop through the entire array. see if you can optimize that after MVP